--- bleach-3.1.0/setup.py.orig	2018-11-09 14:09:24.000000000 +0100
+++ bleach-3.1.0/setup.py	2019-10-09 17:13:57.918597037 +0200
@@ -11,7 +11,7 @@
 setup_requires = []
 if 'test' in sys.argv:
     # Only add pytest-runner to setup_requires if running tests
-    setup_requires.append('pytest-runner>=2.0,<3dev')
+    setup_requires.append('pytest-runner>=2.0')
 
 tests_require = [
     'pytest>=3.0.0',
